backend/src/main/java/gov/cdc/usds/simplereport/service/AzureStorageQueueFhirReportingService.java

```diff
@@ -20,6 +21,9 @@ public final class AzureStorageQueueFhirReportingService implements TestEventRep
   private final QueueAsyncClient queueClient;
   private final GitProperties gitProperties;
 
+  @Value("${simple-report.processing-mode-code:P}")
+  private String processingModeCode = "P";
```

Comment:
In this line we are setting the processingModeCode as P as a safe default value but in truth this is coming from our environment configuration files, right?

Reply:
Great question, if the config exists, it will come from there, otherwise it will use "P" as the default in the @Value() annotation.
The java plain assignment `processingModeCode = "P"` comes handy for plain unit tests that don't spin up a spring context, without that, the @Value() doesn't not do anything, and you would have to use `ReflectionTestUtils.setField()` to set it for that test. I just choose to give it a default java value.
I also tested it and it does function correctly, kind sucks you have to repeat the default value though.